scripts/bootloader/do_sign.py

```diff
@@ -31,9 +32,9 @@ def parse_args():
     return args
 
 
-if __name__ == '__main__':
+if __name__ == 'main':
     args = parse_args()
-    private_key = SigningKey.from_pem(args.private_key.read())
+        private_key = SigningKey.from_pem(args.private_key.read())
     data = args.infile.read()
     signature = private_key.sign(data, hashfunc=hashlib.sha256)
     args.outfile.write(signature)
```

Comment:
- The conditional in the `if` statement should check for `'__main__'` instead of `'main'`. 
  ```suggestion
  if __name__ == '__main__':
  ```
- The indentation of the line `private_key = SigningKey.from_pem(args.private_key.read())` is incorrect and should align with the `args = parse_args()` line. 
  ```suggestion
      private_key = SigningKey.from_pem(args.private_key.read())
  ```